lang/src/org/partiql/lang/eval/physical/window/ExperimentalWindowFunc.kt

```diff
@@ -0,0 +1,5 @@
+package org.partiql.lang.eval.physical.window
+
+@Target(AnnotationTarget.CLASS, AnnotationTarget.FUNCTION, AnnotationTarget.LOCAL_VARIABLE)
+@RequiresOptIn(message = "This API is experimental. It may be changed in the future without notice.", level = RequiresOptIn.Level.ERROR)
+annotation class ExperimentalWindowFunc
```

Comment:
If necessary, we can remove this when merging the feature branch to main.
But for now, if you agree, I would leave it as is since we are only merging this to the feature branch.